homes_guests_loves_styles.css

```diff
@@ -0,0 +1,157 @@
+@import url(fonts.css);
+
+:root {
+    --primary: #3077C6;
+    --primary-dark: #10367C;
+    --secondary-yellow: #F5BD41;
+    --secondary-dark-mode1: #121212;
+    --secondary-dark-mode2: #1D1E1F;
+    --secondary-dark-mode3: #4D4D4D;
+    --primary-stroke: #F3F3F4;
+    --primary-text: #383838;
+    --primary-white: #FFFFFF;
+    --secondary-text: #BFBFBF;
+    --primary-bg: #EAF0F9;
+}
+body {
+    margin: 0;
+    background-color: var(--primary-bg);
+}
+
+h1 {
+    font: 40px Roboto, sans-serif;
+    font-weight: 500;
+    line-height: 46px;
+    text-align: center;
+    margin-top: 120px;
+}
+
+.home_photos_box {
+    padding: 105px 105px 120px 105px;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+
+.home_place_box {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    width: 296px;
+    margin-right: 15px;
+}
+
+.home_photo {
```

Comment:
remember that home photo can be uploaded by any user so it won't be square all the time. You shoud use your knowledge from 'Images & Background" lecure to show only square part of your photo